src/notebooks/telemetry/kernelTelemetry.ts

```diff
@@ -28,7 +30,11 @@ export function sendKernelListTelemetry(
                 counters.kernelLiveCount += 1;
                 break;
             case 'startUsingRemoteKernelSpec':
+                counters.localKernelSpecCount += 1;
+                counters.kernelSpecCount += 1;
```

Comment:
Better telemetry, so we can differentiate between the two
in the past we could only have one or the other not both together.